pre/remap_restart/remap_questions.py

```diff
@@ -121,6 +124,13 @@ def ask_questions():
             "validate": lambda text: validate_geosit_time(text),
             "when": lambda x: x.get("input:shared:GEOS-IT", False) and not x.get("input:shared:MERRA-2", False),
         },
+        {
```

Comment:
Oooh. This one is tricky. By default, GEOSgcm v12 is non-hydrostatic. So, anyone remapping from v12 to v12+ would prefer this to be `True` by default. But then, anyone remapping anything older (v11, say) would want `True`

I am not sure what to do here. @wmputman might have more insight.